packages/react-components/react-message-bar/library/src/components/MessageBarGroup/MessageBarGroup.types.ts

```diff
@@ -18,6 +18,7 @@ export type MessageBarGroupProps = ComponentProps<MessageBarGroupSlots> & {
  */
 export type MessageBarGroupState = ComponentState<MessageBarGroupSlots> &
   Pick<MessageBarGroupProps, 'animate'> & {
+    // TODO: deprecate these 2 obsolete props?
```

Comment:
Yes 👍

Reply:
Done in https://github.com/microsoft/fluentui/pull/33465/commits/ee1613c0bb662ad26468f3643fdd740ebcd0bc69